robottelo/api/utils.py

```diff
@@ -130,6 +133,38 @@ def publish_puppet_module(puppet_modules, repo_url, organization_id=None):
     return cv.read()
 
 
+def import_puppet_module(sat, puppet_module_repo=CUSTOM_PUPPET_MODULE_REPOS['generic_1']):
+    """Download, install and import puppet module. This is workaround for 6.10, there is no longer
+    content view
+
+    :param puppet_module_repo: custom puppet module repository
+    :param sat: Satellite object
+    :return: Puppet environment name,
+        environment name is likely to be searched in next steps of the test
+    """
+    if puppet_module_repo not in list(CUSTOM_PUPPET_MODULE_REPOS.values()):
+        raise ValueError(
+            f'Custom puppet module mismatch, actual custom puppet module repo: '
+            f'"{puppet_module_repo}", does not match any available puppet module: '
+            f'"{list(CUSTOM_PUPPET_MODULE_REPOS.values())}"'
+        )
+    env_name = gen_string('alpha')
```

Comment:
This implies you generate a new environment every time you run this method. Shouldn't it be named `create_puppet_environment`?

Reply:
yes I agree partly, name should be connected to Environment :) 
but we use namings like `entities.Environment().create()`, so in this case better naming can be  `puppet_environment_contains_custom_puppet_repo` or at least `create_custom_environment`.